ansible_rulebook/app.py

```diff
@@ -52,15 +58,15 @@ async def run(parsed_args) -> None:
         ) = await request_workload(
             int(parsed_args.id), parsed_args.websocket_address
         )
+        event_log = asyncio.Queue()
     else:
         inventory = {}
         variables = load_vars(parsed_args)
         rulesets = load_rulebook(parsed_args)
         if parsed_args.inventory:
             inventory = load_inventory(parsed_args.inventory)
         project_data_file = parsed_args.project_tarball
-
-    event_log = asyncio.Queue()
+        event_log = NullQueue()
```

Comment:
@mkanoor We can technically provide the CLI a websocket address but no activation id or worker mode, then a valid asyncio Queue is still needed.